terraform/modules/networking/acm/main.tf

```diff
@@ -9,6 +9,11 @@ terraform {
   }
 }
 
+provider "aws" {
+  alias  = "us_east"
+  region = "us-east-1"
```

Comment:
Is this region intentional?

Reply:
yeah, cloudfront only accepts certificates from us-east-1 unfortunately!